src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -444,6 +444,15 @@ public bool ShowCellOptions()
         return true;
     }
 
+    public override void OnLightLevelChanged(float lightLevel)
+    {
+        // Normalise by maximum light level in the patch
+        var maxLightLevel = Editor.CurrentPatch.GetCompoundAmount("sunlight", CompoundAmountType.Maximum);
```

Comment:
This complex math should be put in the caller. So I suggest the light level is changed to be the following signature:
```
OnLightLevelChanged(float lightFraction, float absoluteLux)
```